.github/workflows/python-app.yml

```diff
@@ -0,0 +1,32 @@
+name: Python application
+
+on:
+  push:
+    branches: [ master, develop ]
+  pull_request:
+    branches: [ master, develop ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.9
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements-dev.txt
```

Comment:
Fix the filename to `requirements_dev.txt`

Reply:
Good catch. Thanks.

```suggestion
        pip install -r requirements_dev.txt
```